src/types/QuantityUnity.ts

```diff
@@ -0,0 +1,5 @@
+export interface QuantityUnity {
```

Comment:
again being pedantic but should be `unit` instead of `unity`

Reply:
I always confuse these both. I changed all of them to Unit